core/runtime/src/main/java/io/quarkus/runtime/TemplateHtmlBuilder.java

```diff
@@ -15,33 +15,112 @@
 public class TemplateHtmlBuilder {
     private static String CSS = null;
 
-    private static final String SCRIPT_STACKTRACE_MANIPULATION = "<script>\n" +
-            "	function toggleStackTraceOrder() {\n" +
-            "		var stElement = document.getElementById('stacktrace');\n" +
-            "		var current = stElement.getAttribute('data-current-setting');\n" +
-            "		if (current == 'original-stacktrace') {\n" +
-            "			var reverseOrder = document.getElementById('reversed-stacktrace');\n" +
-            "			stElement.innerHTML = reverseOrder.innerHTML;\n" +
-            "			stElement.setAttribute('data-current-setting', 'reversed-stacktrace');\n" +
-            "		} else {\n" +
-            "			var originalOrder = document.getElementById('original-stacktrace');\n" +
-            "			stElement.innerHTML = originalOrder.innerHTML;\n" +
-            "			stElement.setAttribute('data-current-setting', 'original-stacktrace');\n" +
-            "		}\n" +
-            "		return;\n" +
-            "	}\n" +
-            "	function showDefaultStackTraceOrder() {\n" +
-            "		var reverseOrder = document.getElementById('reversed-stacktrace');\n" +
-            "		var stElement = document.getElementById('stacktrace');\n" +
-            "       if (reverseOrder == null || stElement == null) {\n" +
-            "           return;\n" +
-            "       }\n" +
-            "		// default to reverse ordered stacktrace\n" +
-            "		stElement.innerHTML = reverseOrder.innerHTML;\n" +
-            "		stElement.setAttribute('data-current-setting', 'reversed-stacktrace');\n" +
-            "		return;\n" +
-            "	}\n" +
-            "</script>\n";
+    private static final String SCRIPT_STACKTRACE_MANIPULATION = """
+            <script>
+                const translations = {
+                    'en': {
+                        'toggleStackTrace': 'Toggle Stack Trace Order',
+                        'defaultStackTrace': 'Default to Reverse Ordered Stack Trace'
+                    },
+                    'pt': {
+                        'toggleStackTrace': 'Alternar Ordem do Stack Trace',
+                        'defaultStackTrace': 'Padrão para Stack Trace em Ordem Reversa'
+                    }
```

Comment:
I'm not entirely sure I want to start this. I could see how problematic it would be if we had a ton of translations and had to update one particular message.

I think the actions are sufficiently easy to understand for us to  not translate them and have a non-localized UI.
It's very technical.